repository/src/main/java/org/apache/atlas/repository/store/graph/v1/DeleteHandlerV1.java

```diff
@@ -180,7 +183,7 @@ public void deleteRelationships(Collection<AtlasEdge> edges, final boolean force
 
                 continue;
             }
-
+            resetHasLineage(edge);
```

Comment:
This method deleteRelationships can also be called from deleteVertex
I fear whenever deleting entity, resetting happening twice
first at AtlasEntityStoreV2.deleteVertices() -> entityGraphMapper.removeHasLineageOnDelete(others);
& 2nd here resetHasLineage(edge)